test/evergreen.yml

```diff
@@ -4169,19 +4169,6 @@ buildvariants:
     - name: compatibility-test-for-newer-releases
     - name: import-compatibility-test
 
-- name: windows-64
-  display_name: "! Windows 64-bit"
-  run_on:
-  - windows-64-vs2017-test
-  expansions:
-    python_binary: 'python'
-    scons_smp_command: -j $(echo "$(getconf NPROCESSORS_ONLN 2>/dev/null || getconf _NPROCESSORS_ONLN 2>/dev/null || echo 1) * 2" | bc)
-  tasks:
-    - name: compile
-    - name: ".windows_only"
-    - name: ".unit_test"
-    - name: fops
-
 - name: windows-64-cmake
```

Comment:
Can we rename `windows-64-cmake` to the new `windows-64` build variant?

Reply:
Yep, sorry for the confusion. This is changed in a later commit (as you discovered in you other comments).